LEGO1/viewmanager/viewroi.h

```diff
@@ -59,7 +60,7 @@ class ViewROI : public OrientableROI {
 	int GetUnknown0xe0() { return m_unk0xe0; }
 	void SetUnknown0xe0(int p_unk0xe0) { m_unk0xe0 = p_unk0xe0; }
 
-	static undefined SetUnk101013d8(undefined p_flag);
+	static MxBool SetLightSupport(MxBool p_flag);
```

Comment:
This part of the code base should not be dependent on Omni and its types. `MxBool` should be `unsigned char` instead